### PR TITLE
Update rooms.json

### PR DESCRIPTION
**Pull request by LEDFlashing**

### CODE REVIEW

File: app/config/tstvhq/rooms.json [hunk 1]
```diff
@@ -102,13 +102,13 @@
         {
             "id": "frontParlor",
             "name": "Front Parlor",
+            "capacity": 10,
+            "description": "Check out these good kids! Sponsor one at woodstocksanctuary.org",
             "type": "IFRAME",
-            "capacity": 100,
-            "muteRoom": true,
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/the-revolt-of-the-angels012green.jpg",
+            "backgroundImage": "./https://tstvhq.s3.amazonaws.com/backgrounds/sleeping porch.jpg",
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:108]: need to get rid of ./

File: app/config/tstvhq/rooms.json [hunk 2]
```diff
@@ -212,14 +212,12 @@
...
+            "type": "JITSI",
             "capacity": 20,
             "muteRoom": false,
             "hideVideo": false,
             "description": "”With a base of amber, sandalwood, tobacco, and frankincense, Baphomet evokes the smokey scent of hidden rooms, velvet thrones, and ancient ritual chambers.“ (-Serpentinae.com) Satanic services are held in this unholy space. ",
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/the-revolt-of-the-angels004green.jpg",
-            "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=ariadnesatanas&parent=estate.tstvhq.com"
+            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/the-revolt-of-the-angels004green.jpg"
             },
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:221]: this bracket now needs to be deleted

File: app/config/tstvhq/rooms.json [hunk 3]
```diff
@@ -382,14 +382,12 @@
...
+            "type": "JITSI",
+            "capacity": 20,
             "muteRoom": false,
             "hideVideo": false,
             "locked": false,
-            "description": "The crushed velvet seats recline! Are they emerald or black? Hard to tell in the dark.",
-            "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=LEDflashing&parent=estate.tstvhq.com"
+            "description": "The crushed velvet seats recline! Are they emerald or black? Hard to tell in the dark."
             },
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:391]: this bracket needs to be deleted

**Review by hndrewaall — CHANGES_REQUESTED** [review]
a few minor issues

File: app/config/tstvhq/rooms.json (hunk 3 above)
Comment (hndrewaall) [reply app/config/tstvhq/rooms.json:389]: here there is definitely an extra brace. it should be deleted, and the comma moved to the end of the previous line

File: app/config/tstvhq/rooms.json [hunk 4]
```diff
@@ -261,13 +258,14 @@
...
-            "description": "Check out these good kids! Sponsor one at woodstocksanctuary.org",
+            "capacity": 100,
+            "muteRoom": true,
+            "hideVideo": false,  
+            "description": "Hang out and chill",
             "type": "IFRAME",
             "backgroundImage": "./https://tstvhq.s3.amazonaws.com/backgrounds/sleeping porch.jpg",
             "iframeOptions": {
-                "src": "https://www.youtube.com/embed/_UItI96xdm4"
-            },
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:270]: this brace shouldn't be deleted, it needs to be below the "src" line

File: app/config/tstvhq/rooms.json [hunk 5]
```diff
@@ -212,15 +212,12 @@
...
+            "type": "JITSI",
             "capacity": 20,
             "muteRoom": false,
             "hideVideo": false,
             "description": "”With a base of amber, sandalwood, tobacco, and frankincense, Baphomet evokes the smokey scent of hidden rooms, velvet thrones, and ancient ritual chambers.“ (-Serpentinae.com) Satanic services are held in this unholy space. ",
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/the-revolt-of-the-angels004green.jpg",
-            "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=ariadnesatanas&parent=estate.tstvhq.com"
-            },
+            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/the-revolt-of-the-angels004green.jpg"
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:220]: this is missing a trailing comma

**Review by hndrewaall — CHANGES_REQUESTED** [review]

File: app/config/tstvhq/rooms.json (hunk 5 above)
Comment (LEDFlashing) [reply app/config/tstvhq/rooms.json]: 'k

File: app/config/tstvhq/rooms.json (hunk 4 above)
Comment (LEDFlashing) [reply app/config/tstvhq/rooms.json:270]: 'k

File: app/config/tstvhq/rooms.json (hunk 3 above)
Comment (LEDFlashing) [reply app/config/tstvhq/rooms.json]: 'k

File: app/config/tstvhq/rooms.json [hunk 6]
```diff
@@ -382,19 +381,15 @@
...
             "hideVideo": false,
             "locked": false,
             "description": "The crushed velvet seats recline! Are they emerald or black? Hard to tell in the dark.",
-            "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=LEDflashing&parent=estate.tstvhq.com"
-            },
             "directions": {
                 "west": "staircase",
-                "north": "belfry"
-            },
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:397]: this brace and comma shouldn't be deleted

Comment (LEDFlashing) [reply app/config/tstvhq/rooms.json:397]: 'k

**Review by hndrewaall — APPROVED** [review]

**Pull request merged**